verify/cockpit-verify

```diff
@@ -22,12 +22,19 @@ mkdir -p mock
 
 echo "Starting testing"
 
+iteration=0
 while true; do
     fail=0
     find cockpit -name '*.py?' -delete
     git -C cockpit fetch origin
     git -C cockpit reset --hard origin/master
-    cockpit/test/vm-download --prune
+    let prune=iteration%25
```

Comment:
The script is `#!/bin/sh`, and this is a bashism. I'd recommend to drop this line and `$prune` altogether, use `$iteration` in the next line, and ..